### PR TITLE
oneMKL migration: Temporary deselection of tests 

### PR DESCRIPTION
**Pull request by Vika-F**
This PR deselects the tests that are failing in oneMKL migration job:
http://intel-ci.intel.com/ef7a857f-11b3-f179-ab7e-a4bf010d0e2d

---

Checklist to comply with before moving PR from draft:

**PR completeness and readability**

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation to reflect the changes or created a separate PR with update and provided its number in the description, if necessary.
- [x] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_.
- [x] I have added a respective label(s) to PR if I have a permission for that.  
- [x] I have resolved any merge conflicts that might occur with the base branch.

**Testing**

- [ ] The unit tests pass successfully.
- [ ] I have run it locally and tested the changes extensively.

**Performance**

- [ ] I have measured performance for affected algorithms using [scikit-learn_bench](https://github.com/IntelPython/scikit-learn_bench) and provided at least summary table with measured data, if performance change is expected.
- [ ] I have provided justification why performance has changed or why changes are not expected.

### CODE REVIEW

**Comment by samir-nasibli** [comment]
@Vika-F please attach the job, that makes the CI green
current branch + OneDAL oneMKL migration branch

**Comment by Vika-F** [comment]
@samir-nasibli I cannot provide the job you are requesting until #2063 is merged. Because the mentioned job would require the changes from #2063.

**Comment by samir-nasibli** [comment]
> @samir-nasibli I cannot provide the job you are requesting until #2063 is merged. Because the mentioned job would require the changes from #2063.

#2063 is merged. Let's check this one

File: onedal/svm/tests/test_nusvr.py [hunk 1]
```diff
@@ -107,7 +107,9 @@ def _test_diabetes_compare_with_sklearn(queue, kernel):
 
 @pass_if_not_implemented_for_gpu(reason="nusvr is not implemented")
 @pytest.mark.parametrize("queue", get_queues())
-@pytest.mark.parametrize("kernel", ["linear", "rbf", "poly", "sigmoid"])
+# TODO: investigate failures in kernel="sigmoid" after migration to oneMKL
```
Comment (samir-nasibli) [inline onedal/svm/tests/test_nusvr.py:110]: please use pytest.skip instead.
See: https://github.com/intel/scikit-learn-intelex/blob/65edc89d26266b89c0948cc9b966129f6b4ac481/sklearnex/ensemble/tests/test_forest.py#L97

**Comment by Alexandr-Solovev** [comment]
Link to the rerun with fixes:  http://intel-ci.intel.com/ef7b39b8-64f0-f1ec-91a0-a4bf010d0e2d

**Review by samir-nasibli — COMMENTED** [review]
Not submitting the block, but please address the comment

**Comment by Vika-F** [comment]
Link to the CI job with the requested changes: http://intel-ci.intel.com/ef7b5b7d-beee-f1af-b697-a4bf010d0e2d
